### ISSUE TITLE
HTML 101 - img without alt attributes

### ISSUE THREAD

**Issue by its-cutie-valerie** [codedex-io/html-101#43]
I've noticed in 06_pet.html and 07_band.html that there are missing "alt" attributes in the images. I was told it's like a "must" so I was wondering about it

**Comment by flameastro** [comment]
@its-cutie-valerie True, the `alt` attribute in the images are really important. Very good observation!
